etc/kayobe/environments/aufn-ceph/a-universe-from-nothing.sh

```diff
@@ -87,7 +87,7 @@ kayobe seed vm provision
 kayobe seed host configure
 
 # Deploy local pulp server as a container on the seed VM
-kayobe seed service deploy --tags seed-deploy-containers --kolla-tags none
+kayobe seed service deploy --tags seed-deploy-containers --kolla-tags none -e deploy_containers_registry_attempt_login=False
```

Comment:
I was hoping we would set this in our config, then add a login task to the pulp container post deploy tasks.

Reply:
Something like this? https://github.com/stackhpc/stackhpc-kayobe-config/pull/891